integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/massindexing/IndexWorkExecutorIT.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.backend.tck.massindexing;
```

Comment:
From the point of view of the backend, "mass indexing" does not exist. It's purely a mapper thing.
Maybe move this to package `org.hibernate.search.integrationtest.backend.tck.work`?